{'description': {'en': 'Personal User Overlay'},
 'feed': ['https://github.com/aliceinwire/overlay/commits/master.atom'],
 'homepage': 'https://github.com/aliceinwire/overlay',
 'name': 'pypy_alice',
 'owner': [{'email': 'alicef@gentoo.org',
            'name': 'Alice Ferrazzi',
            'type': 'person'}],
 'quality': 'experimental',
 'source': [{'type': 'git',
             'uri': 'git://github.com/aliceinwire/overlay.git'},
            {'type': 'git',
             'uri': 'https://github.com/aliceinwire/overlay.git'},
            {'type': 'git', 'uri': 'git@github.com:aliceinwire/overlay.git'}],
 'status': 'unofficial'}
$ pmaint sync pypy_alice
WARNING:pkgcore:repository at location '/home/mgorny/repos/zugaina' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at location '/home/mgorny/repos/zugaina' lacks a defined repo_name
Already up-to-date.
*** syncing pypy_alice
*** synced pypy_alice
 * Sync succeeded
$ pmaint regen --use-local-desc --pkg-desc-index -t 2 pypy_alice
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
 * Cache regenerated successfully
